console/src/components/UIKit/atoms/Spinner/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { StyledSpinner } from './Spinner';
+
+export const Spinner = props => {
+  const { size } = props;
+
+  const spinnerWidth = size === 'small' ? 17 : 20;
```

Comment:
Could you extract `17` and `20` to constants outside of the component?

Reply:
done.